src/test/java/org/broadinstitute/hellbender/tools/HaplotypeCallerSparkIntegrationTest.java

```diff
@@ -0,0 +1,191 @@
+package org.broadinstitute.hellbender.tools;
+
+import org.apache.spark.SparkException;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.engine.AuthHolder;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceWindowFunctions;
+import org.broadinstitute.hellbender.engine.spark.SparkContextFactory;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.GenotypeCalculationArgumentCollection;
+import org.broadinstitute.hellbender.tools.walkers.haplotypecaller.HaplotypeCallerArgumentCollection;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.test.SerializationTestUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+import static org.broadinstitute.hellbender.tools.walkers.haplotypecaller.HaplotypeCallerIntegrationTest.TEST_FILES_DIR;
+import static org.broadinstitute.hellbender.tools.walkers.haplotypecaller.HaplotypeCallerIntegrationTest.calculateConcordance;
+
+public class HaplotypeCallerSparkIntegrationTest extends CommandLineProgramTest {
+
+    /*
+    * Test that in VCF mode we're >= 99% concordant with GATK3.5 results
+    */
+    @Test
+    public void testVCFModeIsConcordantWithGATK3_5Results() throws Exception {
+        Utils.resetRandomGenerator();
+
+        final File output = createTempFile("testVCFModeIsConcordantWithGATK3Results", ".vcf");
+        //Created by running:
+        //java -jar ~/bin/GenomeAnalysisTK-3.5.0/GenomeAnalysisTK.jar -T HaplotypeCaller \
+        // -I ./src/test/resources/large/CEUTrio.HiSeq.WGS.b37.NA12878.20.21.bam \
+        // -R src/test/resources/large/human_g1k_v37.20.21.fasta -L 20:10000000-10100000 \
+        // --out a.gatk3.5.noDownsample.vcf -G StandardHC -G Standard \
+        // --disableDithering --no_cmdline_in_header  -dt NONE --maxReadsInRegionPerSample 100000000 --minReadsPerAlignmentStart 100000
+        final File gatk3Output = new File(TEST_FILES_DIR + "expected.testVCFMode.gatk3.5.vcf");
+
+        final String[] args = {
+                "-I", NA12878_20_21_WGS_bam,
+                "-R", b37_2bit_reference_20_21,
+                "-L", "20:10000000-10100000",
+                "-O", output.getAbsolutePath(),
+                "-pairHMM", "AVX_LOGLESS_CACHING"
+        };
+
+        runCommandLine(args);
+
+        final double concordance = calculateConcordance(output, gatk3Output);
+        Assert.assertTrue(concordance >= 0.99, "Concordance with GATK 3.5 in VCF mode is < 99%");
+    }
+
+    /*
+ * Test that in VCF mode we're >= 99% concordant with GATK3.5 results
+ */
+    @Test(expectedExceptions = SparkException.class) //this should be a UserException, but spark exceptions are not unwrapped yet
```

Comment:
It blows up because allele-specific annotations aren't supported in the base haplotype caller for vcf mode.  It is a copy of a matching test in HaplotypeCallerIntegrationTest.  I've added a comment.